loom/common/utils/utByteArray.h

```diff
@@ -32,6 +32,16 @@ class utByteArray {
     utArray<unsigned char> _data;
     UTsize _position;
 
+    void memcpyAligned(void *destination, const void *source, size_t num)
```

Comment:
I think it could be both, i.e. copy this memory in aligned chunks, so it doesn't throw unaligned errors or copy this unaligned memory in aligned chunks. It's kinda ambigious either way, but maybe unaligned makes more sense at first?